packages/ui-drawer-layout/src/DrawerLayout/DrawerTray/__tests__/DrawerTray.test.js

```diff
@@ -96,17 +99,26 @@ xdescribe('<DrawerTray />', async () => {
   })
 
   it('should apply theme overrides when open', async () => {
-    await mount(
-      <DrawerTray
-        label="DrawerTray Example"
-        open={true}
-        theme={{
+    const themeOverride = {
+      components: {
+        DrawerTray: {
           zIndex: '333'
-        }}
-        render={() => {
-          return 'Hello from layout tray'
-        }}
-      />
+        }
+      }
+    }
+    await mount(
+      <EmotionThemeProvider theme={themeOverride}>
```

Comment:
Normally wrapping it in EmotionThemeProvider is not needed to add override, would be enough to add `themeOverride={{}}` directly on the component, but this fails here because of another reason. We need to omit "themeOverride" prop in the `omitProps` handler, so it doesn't get passed through: I'll fix it on the main branch.

Reply:
nice find, I'll fix it :)